### DISCUSSION TITLE
I need a refund

### DISCUSSION THREAD

**Discussion by wei2507102513** [aclap-dev/video-downloadhelper#1130] [License & Purchase requests]
### What's your issue?

My license is tied to Chrome, but I find that I can't download YouTube videos. I need a refund

### The 10 first digits (only 10) of your license number or email address

0e8c7b71a3

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
